props/saber_fett263_buttons.h

```diff
@@ -1573,6 +1573,30 @@ SaberFett263Buttons() : PropBase() {}
     }	  
   }
 
+  bool CheckInteractivePreon() {
+    bool includes_interactive_preon;
```

Comment:
First of all, variables like these should be initialized, otherwise they will have a random value, so like this:

   bool include_interactive_preon = false;

However, you don't need this variable. Instead of doing

    includes_interactive_preson |= CONDITION;

you can just do:
    if (CONDITION) return true;

and then end the function with:
   return false;

Reply:
Ah, thanks.  OK simplified both of these functions and moved to prop_base